src/main/java/training/commands/ArmCommands/elbowCommands/ElbowDefultCommand.java

```diff
@@ -0,0 +1,26 @@
+package training.commands.ArmCommands.elbowCommands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.Subsystem;
+import training.subsystems.ArmSubsystems.elbowSubsystem.Elbow;
+import training.subsystems.ArmSubsystems.elbowSubsystem.neoElbow.NeoElbow;
+
+public class ElbowDefultCommand extends Command {
```

Comment:
```suggestion
public class ElbowDefaultCommand extends Command {
```
also i would not call this default command, because the name says usage and not what the command is doing